.github/workflows/linter-cicd.yml

```diff
@@ -75,6 +75,9 @@ jobs:
       uses: actions/setup-python@v4
       with: { python-version: "3.11" }
 
+    - name: Install Python Developer Headers
+      run: sudo apt install python3.11-dev
```

Comment:
Uhh don't do this use the install Python action they have an option for headers iirc

Reply:
ok will check